### PR TITLE
Layout file modified

### PR DESCRIPTION
**Pull request by dasbeerboot**

### CODE REVIEW

**Comment by oudwud** [comment]
Basically, I highly recommend to use `*.md` file in git, instead of `*.txt` file.
You can use nice syntax and auto-generated web view for `*.md` files. (like `README.md`)

**Comment by oudwud** [comment]
Good job. Now you pushed two changes (deleting a file and creating new file).
Fortunately, `git` recognized that the file was logically renamed.
Because your `*.md` file is not following markdown syntax, it is not being displayed pretty in markdown format.
You need to be comfortable to write markdown files in the future.

File: layout.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Main theme for all pages = Background: Black, Letters: White, Font: idk
+If applicant puts right answer -> Go to next pages
+->If not, Go to "Wrong answer page"
```
Comment (stevechun) [inline layout.md]: Please add a space after ->

File: layout.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+X (sign)
+You have chosen a wrong answer
+[Try again] (button)
+
+#2
+
+Solve the following question, and convert the answer into MD5 hash and submit the result
+(*No space, all alphabets must be in lower case)
+
+Q: What is the name of the amusement park you went with Juwon for the first time in Korea?
```
Comment (makeajourney) [inline layout.md]: You've told me that you were not gonna make a question about you and your boyfriend!
Such a liar!

File: layout.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+Main theme for all pages = Background: Black, Letters: White, Font: idk
+If applicant puts right answer -> Go to next pages
+->If not, Go to "Wrong answer page"
+
+
+
+#1
```
Comment (oudwud) [inline layout.md]: In markdown syntax, `#` means a section. You need to add a space after `#`.
```
# 1
```
or
```
# Plan 1
```

In addition, you can add a subsection and a sub-subsection using `##` and `###`.
```
## Subsection of Plan 1

blah blah

### Sub-subsection of Plan 1
```

File: layout.md (hunk 2 above)
Comment (stevechun) [reply layout.md]: Also, please make it clear that it is `South`, not `North`

Comment (dasbeerboot) [reply layout.md]: You all can die. I hate y'all

File: layout.md (hunk 1 above)
Comment (dasbeerboot) [reply layout.md]: Respect my space please

File: layout.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+#3
+https://www.mathsisfun.com/puzzles/easter-eggs-eggsactly-solution.html
+
+1. $1
+2. $1.5
+3. $2
+4. $2.5
```
Comment (oudwud) [inline layout.md]: If you want to emphasize something with different format, you can wrap them in three-backticks.
Screenshot from 2019-03-25 22-53-07

Then, Github will display it as below.
```
1. $1
2. $1.5
3. $2
4. $2.5
```

File: layout.md (hunk 2 above)
Comment (dasbeerboot) [reply layout.md]: And officially it's republic of korea man

File: layout.md (hunk 1 above)
Comment (oudwud) [reply layout.md]: You can click the `Resolve conversation` button if you applied this feedback (or if you want to ignore it).

File: layout.md (hunk 4 above)
Comment (dasbeerboot) [reply layout.md]: Waaaaaaa sugoi danke

File: layout.md (hunk 2 above)
Comment (makeajourney) [reply layout.md]: There's no official. How can I know it's a republic or not?

File: layout.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Exam agreement: 
+You must read this form and check the box at the bottom before you start the exam. Please ask the invigilator if there is anything you are not sure about.
+
+* Information about the duration of the exam will be given in advance.
+* Candidate will not be allowed to use any search engine during the exam. Any evidence of internet use outside of this website will be investigated as an attempt to cheat.
+* Candidate may have invigilator by his side.
+* Candidate may have one chance to ask for hint during the exam and this shall be personally advised by invigilator.
+* If there are any queries regarding the exam questions the candidate must ask the invigilator.
+
+ㅁ I have read and understood the above
```
Comment (oudwud) [inline layout.md]: Please remove a `ㅁ` if it's a typo. If it was intended, please click the `Request a re-review` button for me.
image

File: layout.md (hunk 4 above)
Comment (makeajourney) [inline layout.md]: I want to recommend you that separate the design concept and data from your layout file.
But it shouldn't be done with this PR. You can be done with the next PR. 
It's just my opinion. I think the most important thing as a beginner is trying to have the implementation of your design or what you've thought to want to make. So you should not know all like git or markdown. Just pay your attention to make the web.

**Comment by makeajourney** [comment]
I want to recommend you that separate the design concept and data from your layout file.
But it shouldn't be done with this PR. You can be done with the next PR. 
It's just my opinion. I think the most important thing as a beginner is trying to have the implementation of your design or what you've thought to want to make. So you should not know all like git or markdown. Just pay your attention to make the web.

File: layout.md (hunk 2 above)
Comment (dasbeerboot) [reply layout.md]: South Korea’s official name is Republic of Korea and it’s a common sense

Comment (dasbeerboot) [reply layout.md]: https://simple.m.wikipedia.org/wiki/South_Korea

**Pull request merged**